app/controllers/playbook/pages_controller.rb

```diff
@@ -64,5 +65,12 @@ def set_kit
         redirect_to root_path, flash: { error: "That kit does not exist" }
       end
     end
+
+    def ensure_rails_kit_exists
+      # TODO: unsure why we cannot simply use the helpers that are included in ApplicationController - fix this
+      erbfiles = File.join("**", "*.erb")
+      kit_files = Dir.glob(erbfiles, base: "#{Playbook::Engine.root}/app/pb_kits/playbook/pb_#{@kit}/docs").present?
+      redirect_to action: "kit_show_react" unless kit_files.present?
```

Comment:
Since we default to `rails` kit version from the nav, we should redirect if that doesn't exist (small edge case, but relevant)